### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by zainsadaqat**
In this pull request, I Implemented the following features:

- Use Gitflow
- Created project using Webpack
- Style layout for the leaderboard project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,13 +13,15 @@
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.24.2",
     "hint": "^6.1.4",
+    "html-loader": "^2.1.2",
```
Comment (hemantso) [inline package.json:17]: - [x] Script like `start` is missing in your package.json. Kindly try to add the correct script to start the project locally.

Webpack configured script
image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @zainsadaqat , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (zainsadaqat) [reply package.json:17]: Successfully Added the start script in the **package.json** file

**Review by henatan99 — APPROVED** [review]
Hi @zainsadaqat , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] OPTIONAL: Please, update your readme. You kept the default Microverse readme template without updating it. It is important and it should have been pointed out by the previous reviewer. Kindly make this change even though it in not required for your project's submission. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zainsadaqat** [comment]
### @hemant-soni-vst-au4 Thanks for pointing out this mistake. I completely forgot to update my readme. 
I'm going to update it now.

**Pull request merged**